build/common/init_command.sh

```diff
@@ -82,11 +85,13 @@ elif [ "$MODULE_NAME" = 'celery_monitoring' ]; then
         echo "Starting the 'monitoring' celery worker"
         celery -A openwisp worker -l ${DJANGO_LOG_LEVEL} --queues monitoring \
                -n monitoring@%h --logfile /opt/openwisp/logs/celery_monitoring.log \
-               --pidfile /opt/openwisp/celery_monitoring.pid --detach
+               --pidfile /opt/openwisp/celery_monitoring.pid --detach \
+               ${OPENWISP_CELERY_MONITORING_COMMAND_FLAGS}
         echo "Starting the 'monitoring_checks' celery worker"
         celery -A openwisp worker -l ${DJANGO_LOG_LEVEL} --queues monitoring_checks \
                -n monitoring_checks@%h --logfile /opt/openwisp/logs/celery_monitoring_checks.log \
-               --pidfile /opt/openwisp/celery_monitoring_checks.pid --detach
+               --pidfile /opt/openwisp/celery_monitoring_checks.pid --detach \
+               ${OPENWISP_CELERY_MONITORING_COMMAND_FLAGS}
```

Comment:
I think here we should have a dedicated variable: `OPENWISP_CELERY_MONITORING_CHECKS_COMMAND_FLAGS`, so we can control the amount of workers for both queues, it's not 100% sure they have to be always the same.